src/misc/helper.ts

```diff
@@ -850,7 +909,7 @@ export const verifyBlock = (block: Block): Array<number | boolean> => {
 
 export const verifyReward = (block: Block): number[] => {
   const response = [];
-  let reward = 0;
+  let reward = .0;
```

Comment:
What is this format supposed to return? `0.1`? We should document this 👍

Reply:
.0 = 0.0 Just to document those values can be floating numbers.

Reply:
.1 would be 0.1

Reply:
I think we should add this as a comment as well so nobody gets confused about this 👌

Reply:
Okay. I guess this is just a strange habit I developed using my CAS-Calculator...

Reply:
fixed with 968219f